### PR TITLE
fix(Accordion): Correctly infer icon style from indicatorPosition="left"

### PR DESCRIPTION
**Pull request by lukelooker**
Also changes `Fieldset` icon size from `medium` to `small` to be consistent with `Accordion`

## Developer Checklist [ℹ️](https://github.com/looker-open-source/components/blob/main//CONTRIBUTING.md#developer-checklist)

- [x] 🖼 Image Snapshot coverage
- [x] 📚 Documentation updated
- [ ] 👾 Browsers tested
  - [x] Chrome
  - [x] Firefox
  - [x] Safari

### CODE REVIEW

File: packages/components/src/Accordion/accordionDefaults.tsx [hunk 1]
```diff
@@ -44,7 +46,7 @@ export interface AccordionContextProps {
   onOpen?: () => void
 }
 
-export const accordionContextDefaults: AccordionContextProps = {
+export const accordionDefaults: AccordionDefaultProps = {
```
Comment (ElliotTheHuman) [inline packages/components/src/Accordion/accordionDefaults.tsx:49]: I think we're safe to scrap the control-related props from `accordionDefaults` (and from the interface).

These don't have a "default" per say, but some of them previously had default values for typing purposes.

Comment (ElliotTheHuman) [inline packages/components/src/Accordion/accordionDefaults.tsx:49]: I think we're safe to scrap the control-related props from accordionDefaults (and from the interface).

These don't have a "default" per say, but some of them previously had default values for typing purposes.

**Review by ElliotTheHuman — CHANGES_REQUESTED** [review]
Just a minor comment about the `AccordionContext => accordionDefaults` conversion. Outside of that, looks shway!

File: packages/components/src/Accordion/accordionDefaults.tsx (hunk 1 above)
Comment (lukelooker) [reply packages/components/src/Accordion/accordionDefaults.tsx:43]: Ah yes! Fixed! :)

**Review by ElliotTheHuman — APPROVED** [review]
Shway!

**Pull request merged**